tensorflow_datasets/audio/librispeech.py

```diff
@@ -199,7 +199,7 @@ def _generate_librispeech_examples(directory):
   transcripts_glob = os.path.join(directory, "LibriSpeech", "*/*/*/*.txt")
   for transcript_file in tf.io.gfile.glob(transcripts_glob):
     path = os.path.dirname(transcript_file)
-    with tf.io.gfile.GFile(os.path.join(path, transcript_file)) as f:
```

Comment:
Could you split this PR in 2 (one for the new dataset and one for the librispeech.py fix) ?

Also I think it still works even without the fix as `transcript_file` is an absolute path and `os.path.join('/a/b/c', '/d/e/f') == '/d/e/f'`